inc/views/base/warnings/warning_row.twig

```diff
@@ -0,0 +1,31 @@
+{% set row = alt_trow() %}
```

Comment:
@justinsoltesz was talking about completely removing alternating row classes. I've left them in the templates I've done too, but we might end up removing them (and the function) completely...